<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newCircuito</class>
 <widget class="QDialog" name="newCircuito">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Novo circuito</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="entradasSpinBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="entradasLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Número de entradas:</string>
   </property>
  </widget>
  <widget class="QLabel" name="saidasLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Número de saídas:</string>
   </property>
  </widget>
  <widget class="QLabel" name="portasLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Número de portas:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="saidasSpinBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="portasSpinBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>newCircuito</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>newCircuito</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
